docs/deployment/README.md

```diff
@@ -147,298 +137,208 @@ These resources only need to be created once per account.
+Copy the [`terraform.tfvars.example`](https://github.com/nasa/cumulus-template-deploy/blob/master/data-persistence-tf/terraform.tfvars.example) file to `terraform.tfvars`, and fill in
+appropriate values. See the [data-persistence module variable definitions](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/variables.tf) for more detail on each variable.
 
-The `iam` deployment also creates an instance profile named `<stack-name>-ecs` that can be viewed from the AWS CLI command: `aws iam list-instance-profiles`.
+**Reminder:** Elasticsearch is optional and can be disabled using `include_elasticsearch = false` in your `terraform.tfvars`.
 
-### Deploy the Cumulus database stack
+Run `terraform init` if this is the first time deploying this module. You should see output like:
 
-This section will cover deploying the DynamoDB and ElasticSearch resources.
-Reminder: ElasticSearch is optional and can be disabled using `es: null` in your `config.yml`.
+```shell
+* provider.aws: version = "~> 2.32"
 
-**Deploy `db` stack**
+Terraform has been successfully initialized!
+```
 
-```bash
-  $ DEPLOYMENT=<cumulus-deployment-name> \
-      AWS_REGION=<region> \ # e.g. us-east-1
-      AWS_PROFILE=<profile> \
-      npm run deploy-db
+Run `terraform apply` to deploy your data persistence resources. Type `yes` when prompted to confirm that you want to create the resources. Assuming the operation is successful, you should see output like:
+
+```shell
+Apply complete! Resources: 16 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+dynamo_tables = {
+  "access_tokens" = {
+    "arn" = "arn:aws:dynamodb:us-east-1:12345:table/prefix-AccessTokensTable"
+    "name" = "prefix-AccessTokensTable"
+  }
+  # ... more tables ...
+}
+elasticsearch_alarms = [
+  {
+    "arn" = "arn:aws:cloudwatch:us-east-1:12345:alarm:prefix-es-vpc-NodesLowAlarm"
+    "name" = "prefix-es-vpc-NodesLowAlarm"
+  },
+  # ... more alarms ...
+]
+elasticsearch_domain_arn = arn:aws:es:us-east-1:12345:domain/prefix-es-vpc
+elasticsearch_hostname = vpc-prefix-es-vpc-abcdef.us-east-1.es.amazonaws.com
+elasticsearch_security_group_id = sg-12345
 ```
 
+Your data persistence resources are now deployed.
 
-### Deploy the Cumulus application stack
+### Deploy the Cumulus Message Adapter layer
 
-This section will cover deploying the primary Cumulus stack, containing compute resources, workflows and all other AWS resources not covered in the two stacks above.
+**Note:**: If you are deploying in an NGAP environment, you should be able to use the existing Cumulus Message Adapter layer deployed in your environment.
 
-Once the preceding configuration steps have completed, run the following to deploy Cumulus from your `<daac>-deploy` root directory:
+The [Cumulus Message Adapter (CMA)](./../workflows/input_output.md#cumulus-message-adapter) is necessary for interpreting the input and output of Cumulus workflow steps. The CMA is now integrated with Cumulus workflow steps as a Lambda layer.
```

Comment:
Added CMA deployment instructions

Reply:
This looks good.    Annoying that users are going to have to do it though.